### PR TITLE
Add a rust article aggregator site to Resources

### PR DESCRIPTION
**Pull request by danloh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1450,6 +1450,7 @@ A registry allows you to publish your Rust libraries as crate packages, to share
 * [Rust Guidelines](http://aturon.github.io/)
 * [RustBooks](https://github.com/sger/RustBooks) — list of RustBooks
 * [Rust Subreddit](https://www.reddit.com/r/rust/) — A subreddit(forum) where rust related questions, articles and resources are posted and discussed
+* [Toplog](https://toplog.cc/) — an Article/Book/Event/Talk Aggregator for Rust by Rust
```
Comment (palfrey) [inline README.md]: I'm seeing a lot of Rust feeds, but no Rust code for the site. Is there source code available?

Comment (danloh) [reply README.md]: yes:  https://github.com/danloh/toplog ,  just push to GitHub

**Comment by palfrey** [comment]
I'm not really sure with this one. The aggregator itself isn't Rust specific (although >50% of the current feeds are for Rust people), and the source code has no README, no license, no CI and 1 commit so far. It feels like a very brand new project, and I'd like to see it get a bit further before it gets considered for addition here.

@luciusmagn What do you think?

**Comment by danloh** [comment]
Ok,  it's fine, I can close this

**Pull request closed without merging**